include/battle.h

```diff
@@ -202,17 +202,17 @@ struct SpecialStatus
     u8 lightningRodRedirected:1;
     u8 restoredBattlerSprite: 1;
     u8 faintedHasReplacement:1;
-    u8 focusBanded:1;
-    u8 focusSashed:1;
     u8 emergencyExited:1;
     u8 afterYou:1;
+    u8 enduredDamage:1;
+    u8 unused_1:1;
```

Comment:
Any reason we don't just group these as three unused bits instead of keeping the byte separation as it currently is?